### ISSUE TITLE
Wrong System Settings icon?

### ISSUE THREAD

**Issue by D3SOX** [bikass/kora#103]
Hi,

since the new release 1.4.0 the icon for the KDE System Settings is now (I assume) wrong:
![dock](https://user-images.githubusercontent.com/24937357/104429890-62699f80-5586-11eb-8282-f02add13211e.png)

However in my Window Title applet it is still the old one:
![applet](https://user-images.githubusercontent.com/24937357/104429994-79a88d00-5586-11eb-8465-aceda248b0dd.png)

You can also see this under the icons section in the settings:
![icons](https://user-images.githubusercontent.com/24937357/104430131-9644c500-5586-11eb-8532-59066d9d5892.png)

I'm not sure but I think this was introduced in https://github.com/bikass/kora/commit/9370fc8d0364a8ac76fa21b8f1045a52e4353ccd
It uses `preferences-system`
![desktopentry](https://user-images.githubusercontent.com/24937357/104431248-f8ea9080-5587-11eb-972f-7c4d2ec7b385.png)
Perhaps it uses `actions/scalable/preferences-system.svg` which leads to `actions/scalable/system-run.svg` and I think this is the icon I see

Sorry if this is intended.

**Comment by bikass** [comment]
Hi, I'm not Plasma user at the moment, but I assume it should be icon from apps(blue) and not from actions.
If you have time, could you try changing in index actions MaxSize to something lower, lets say 24?

**Comment by D3SOX** [comment]
It is already at 16
![index](https://user-images.githubusercontent.com/24937357/104437508-2424ae00-558f-11eb-9f73-77afdff274ae.png)

**Comment by bikass** [comment]
Yes but Max is 512. I'm not sure if it will work just worth of trying.

**Comment by D3SOX** [comment]
Changing every instance of `MaxSize=16` to `MaxSize=24` hasn't changed anything

**Comment by bikass** [comment]
@Digitalone1 I think we need to look at index for these issues. When I changed type to Fixed from scalable in actions and removed Min Max it fixed some issues on KDE.

**Comment by Digitalone1** [comment]
There are two `preferences-system`, one in **apps** and one in **actions**.

KDE is picking the one in actions rather than the correct one in apps.

Before it was
```
[actions/scalable]
Context=Actions
Size=16
Type=Fixed

[actions/scalable@2]
Context=Actions
Size=16
Scale=2
Type=Fixed
```

Anyway, if this issue is happening only in the icon chooser of the system settings, and the icon is good in the app itself and the menu list, I think it's not a big issue.

Otherwise we should return to the previous 16 fixed specification in the index, and maybe rename the folder from scalable to 16, like for panel icons.

@D3SOX 

Try to change the lines as follows and see if something changes after reloading the icon theme (you may need to close and restart the Plasma session to see changes):
```
[actions/scalable]
Context=Actions
Size=16

[actions/scalable@2]
Context=Actions
Size=16
Scale=2
```

**Comment by bikass** [comment]
Yes, I tried in VM and it worked. Now it's picking up action icon in places where it should be from apps. I will fix index later.

**Comment by Digitalone1** [comment]
So it's good like this since they are not `Fixed`, but we can't use `Scalable`. If type it's not specified, it defaults to `Threshold`.

**Comment by bikass** [comment]
I Min Max and changed Type=Fixed. Of-course needs some testing now.

**Comment by Digitalone1** [comment]
@bikass Please, test it on VM without the line fixed Type for `actions/scalable` and `actions/scalable@2`. I'd like to know if it's the same.

**Comment by bikass** [comment]
@D3SOX  yes, now looks everything ok.
@Digitalone1 I tested on Kubuntu, Xubuntu, Solus and Mint(Cinamon). I haven't seen any changes without Type=Fixed. Checked other themes, they also have - Fixed.

Offtopic. While testing these systems, looks like just Cinamon has Hidpi support (sort of x2). And Solus way faster then others.

**Comment by Digitalone1** [comment]
Since they're fixed like panels, we could change the directory from scalable to 16 because it can be confusing.

**Comment by bikass** [comment]
Yes, why not. Would be cleaner.

**Comment by Digitalone1** [comment]
I'm doing it.

**Comment by Digitalone1** [comment]
So I made some changes. Folder 16 actions in place of scalable. 

Copied some things from breeze icon theme. One is interesting for categories:
```
#32x32 - Fixed size - For categories icons >!!!ONLY!!!< - Used in Kickoff (KDE 4.x.x) and Lancelot. Also used in MATE and Cinnamon (just FYI)
```

So categories scalable icons seem to be needed for other DE. I made a specific folder categories scalable copying the one in breeze categories with symlink to icons already present in kora app/scalable.

Then in Adwaita there are few categories symbolic icons missing in kora categories symbolic. Please @bikass, add the remaining:

![Schermata del 2021-01-13 17-24-22](https://user-images.githubusercontent.com/25790525/104482282-0d9a4900-55c7-11eb-9a4f-a030fa8e8139.png)

I'm going to test on my system.

**Comment by bikass** [comment]
Ok, thanks,
Made missing Categories/symbolic.

**Comment by Digitalone1** [comment]
Great. Now it's more complete and cleaner.

Another thing I'd like to test, but not now, maybe after the next release, it's the `Inherits` key.

From the [specification](https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html#file_formats) it's defines as follows:

> The name of the theme that this theme inherits from. If an icon name is not found in the current theme, it is searched for in the inherited theme (and recursively in all the inherited themes).

Just like a breeze icon not present in kora, but the system shows it the same because breeze is in `Inherits` list. This way should be possible to specify `kora` as the first in the Inherits list for kora light and delete all the symlinks in `kora-light` folders.

Should be tested, but in the future. It's okay now.

**Comment by bikass** [comment]
Oh, that is interesting!

**Comment by Digitalone1** [comment]
> Oh, that is interesting!

Sure, if it's working it's a great improvement because we can delete all symlinks in the theme root folder.

I remember to have already tried to do it, but failed, maybe because I specified theme name in the inherits while I think theme folder name should be provided, so `kora` and not Kora, `kora-light` and not Kora Light... 

Anyway, I suggest to test this only after the next release.

**Comment by Digitalone1** [comment]
@bikass I think a new release con be made to fix this Plasma issue and push the new configuration.
